tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -60,46 +62,79 @@ void test_binary_compact_object_construction(
+  constexpr double xcoord_objectB = 3.0;
 
   // Enveloping Cube:
-  const double radius_enveloping_cube = 25.5;
-  const double radius_enveloping_sphere = 32.4;
+  constexpr double radius_enveloping_cube = 25.5;
+  constexpr double radius_enveloping_sphere = 32.4;
 
   // Misc.:
-  const size_t refinement = 2;
-  const size_t grid_points = 6;
+  constexpr size_t refinement = 2;
+  constexpr size_t grid_points = 6;
+  constexpr bool use_projective_map = true;
+
+  // Options for outer sphere
+  constexpr size_t outer_layer_radial_refinements = 3;
 
   for (const bool excise_interiorA : {true, false}) {
     for (const bool excise_interiorB : {true, false}) {
       for (const bool use_equiangular_map : {true, false}) {
-        const domain::creators::BinaryCompactObject binary_compact_object{
-            inner_radius_objectA,
-            outer_radius_objectA,
-            xcoord_objectA,
-            excise_interiorA,
-            inner_radius_objectB,
-            outer_radius_objectB,
-            xcoord_objectB,
-            excise_interiorB,
-            radius_enveloping_cube,
-            radius_enveloping_sphere,
-            refinement,
-            grid_points,
-            use_equiangular_map};
-        test_binary_compact_object_construction(binary_compact_object);
+        for (const bool use_logarithmic_map_outer_spherical_shell :
+             {true, false}) {
+          const domain::creators::BinaryCompactObject binary_compact_object{
+              inner_radius_objectA,
+              outer_radius_objectA,
+              xcoord_objectA,
+              excise_interiorA,
+              inner_radius_objectB,
+              outer_radius_objectB,
+              xcoord_objectB,
+              excise_interiorB,
+              radius_enveloping_cube,
+              radius_enveloping_sphere,
+              refinement,
+              grid_points,
+              use_equiangular_map,
+              use_projective_map,
+              use_logarithmic_map_outer_spherical_shell,
+              outer_layer_radial_refinements};
+          test_binary_compact_object_construction(binary_compact_object);
+
+          // Also check whether the radius of the inner boundary of Layer 5 is
+          // chosen correctly.
+          // Compute the radius of a point in the grid frame on this boundary.
+          // Block 44 is one block whose -zeta face is on this boundary.
+          const auto map{binary_compact_object.create_domain()
+                             .blocks()[44]
+                             .stationary_map()
+                             .get_clone()};
+          tnsr::I<double, 3, Frame::Logical> logical_point(
+              std::array<double, 3>{{0.0, 0.0, -1.0}});
+          const double layer_5_inner_radius = get(
+              magnitude(std::move(map)->operator()(std::move(logical_point))));
+          const double divisions_in_outer_layers =
+              static_cast<double>(pow(2, outer_layer_radial_refinements) + 1);
```

Comment:
Line 123 doesn't take into account the initial refinement, so at first glance `divisions_in_outer_layers` appears to be a misnomer - this is the number of layers that you want though, maybe a comment saying
`// Total number of divisions in layers 4 and 5, modulo initial refinement` or something else that conveys that it is only the relative refinement that needs to be taken into account. (Because `divisions_in_outer_layers` isn't really the total number of divisions in the outer layer.)